### PR TITLE
Fix compass no network issue

### PR DESCRIPTION
**Pull request by OmerGery**

### CODE REVIEW

**Review by avivbarniv — COMMENTED** [review]
who changes it to false atm?

**Comment by OmerGery** [comment]
onRegionChangeEvent - triggered both after compass and after drag [here](https://github.com/Autofleet/locomotion/blob/17c6195a6c5570cf2e281d97bc1360702856ac4b/examples/client/Locomotion/src/pages/ActiveRide/newMap.js#L330)  only after network is available.

**Review by avivbarniv — APPROVED** [review]
change base branch

**Review by sephina — APPROVED** [review]

**Review by avivbarniv — CHANGES_REQUESTED** [review]
Why is it part of the context?
the setpickup is a local variable that scoped only to the activeRide page, it shouldn't be part of the context

File: examples/client/Locomotion/src/pages/ActiveRide/index.js [hunk 1]
```diff
@@ -392,6 +413,11 @@ const RidePage = ({ mapSettings, navigation }) => {
       }
     }
   }, [isExpanded]);
+  useEffect(() => {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/ActiveRide/index.js:417]: remov eit

File: examples/client/Locomotion/src/pages/ActiveRide/index.js [hunk 2]
```diff
@@ -255,6 +263,20 @@ const RidePage = ({ mapSettings, navigation }) => {
     ),
     [BS_PAGES.ACTIVE_RIDE]: () => <ActiveRide />,
   };
+  const updateLocationOnMapData = async (lat, lng) => {
+    const spData = await reverseLocationGeocode(lat, lng);
+    if (spData) {
+      saveSelectedLocation(spData);
+      setPickupChanged(true);
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/ActiveRide/index.js:271]: setPickupChanged(currentBsPage === BS_PAGES.CONFIRM_PICKUP)

**Review by avivbarniv — APPROVED** [review]

**Pull request merged**